fusion_docs/index.md

```diff
@@ -39,6 +39,13 @@ When pipelines run with cloud storage, tasks typically read data from a bucket,
 
 The result is significant overhead for every task. Fusion enables direct file access to cloud object storage, eliminating unnecessary I/O and dramatically reducing data movement and overall runtime.
 
+### Checkpoint and recovery terminated instances
+
+Running pipelines on AWS Spot instances can lead to interruptions, causing tasks to restart from the beginning and waste time and compute resources.
+
+Fusion Snapshots enable checkpointing of running Nextflow processes, allowing tasks to be resumed on a different machine if interrupted.
+By leveraging the Fusion file system and shared S3 storage, tasks seamlessly continue from the last checkpoint, improving efficiency and reducing resource usage.
```

Comment:
Not sure it really is because of that? I'd maybe phrase it more about how Fusion can freeze running processes and save their state to s3 so that they can be resumed on another instance at a later date from the exact same CPU cycle, without losing anything.

Reply:
Rephrased. I'm using checkpoint rather than freeze to keep terminology consistent.